registry/configuring-registry-operator.adoc

```diff
@@ -1,4 +1,4 @@
-[id=configuring-registry-operator’]
+[id="configuring-registry-operator"]
 = Image Registry Operator in {product-title}
 include::modules/common-attributes.adoc[]
 :context: configuring-registry-operator
```

Comment:
I think the following note needs to be updated: 

Storage is only automatically configured when you install on Amazon Web Services.

Reply:
Storage for the registry is automatically configured when doing an IPI install on AWS, GCP, or Azure.